### ISSUE TITLE
multiplayer error in tippers.lua

### ISSUE THREAD

**Issue by Satissis** [netjungle/courseplay#57]
I was testing something in multiplayer and when I setup a tipper to run under the silo and goto the tip spot right next to the silo (this was a test and I think this will apply to all tip triggers)
it throws these 2 errors while unloading for the first time.

```
Error: LUA running function 'update'
C:/Users/Satis/Documents/My Games/FarmingSimulator2013/mods/ZZZ_courseplay/tippers.lua(279) : attempt to perform arithmetic on field 'loopCounter' (a nil value)
Error: LUA running function 'update'
C:/Users/Satis/Documents/My Games/FarmingSimulator2013/mods/ZZZ_courseplay/tippers.lua(275) : attempt to compare number with nil

```

The error do not show on the host but only on the joined players.

These error will continue printing until the trailer is empty and the tractor + tipper drives away again.

The next time it tries to dump at the tip trigger again, no error is shown.

I looked a bit in the tippers.lua and what I notised, then it doesn't appear that the:
self.loopCounter is set the first time it runs.
A fix for this would be to define the self.loopCounter before using it and maybe send it in the client join streamWrite and streamRead event so it won't throw this error again.

Kind regards
Satis

**Comment by JakobTischler** [comment]
Well this is embarrassing. We clearly forgot to streamWrite/read it. (Hopefully) fixed in commit [428cf7b80d](https://github.com/netjungle/courseplay/commit/428cf7b80d0b62cc34d1a9018b9d6cbb5533b4d3).

**Comment by Satissis** [comment]
I'll close this one now since the loopCounter have been removed in commit https://github.com/netjungle/courseplay/commit/f8d0eb4f940de79a9402d45288517e62bcef9eb8
